extensions/console/02_console.js

```diff
@@ -195,8 +195,8 @@
       const needed = (columnWidths[i] - len) / 2;
       // round(needed) + ceil(needed) will always add up to the amount
       // of spaces we need while also left justifying the output.
```

Comment:
Per [Simon's observation](https://github.com/denoland/deno/pull/11295/files#r664785314) (1/2 — _[Code change suggestions don't currently allow for including deleted lines](https://github.com/isaacs/github/issues/1755), so this is continued in the next suggestion._)

```suggestion
      const needed = columnWidths[i] - len;
```